### PR TITLE
Add to long position check take profit and do action with TP when order was cancel or close position

### PR DESCRIPTION
**Pull request by vlad-sundunchan**

### CODE REVIEW

**Comment by markblaize** [comment]
Task linked: [CU-861meb3k7 Add to long position check take profit and do action with tp when order was close position](https://app.clickup.com/t/861meb3k7)

**Comment by Pavel-Blaize** [comment]
Task linked: [CU-861meb3k7 Add to long position check take profit and do action with tp when order was close position](https://app.clickup.com/t/861meb3k7)

File: contracts/leverage_trading/src/metadata.rs [hunk 1]
```diff
@@ -89,6 +90,8 @@ pub struct Order {
     pub block: BlockHeight,
     pub timestamp_ms: Timestamp,
     pub lpt_id: String,
+    /// data after closed position for trade history -> (Fee, PnL)
+    pub data_for_position_history: Option<(U128, PnLView)>,
```
Comment (TymofiiSavinov) [inline contracts/leverage_trading/src/metadata.rs:94]: Rename it to history_data and create separate structure for it with fields fee, pnl

**Review by TymofiiSavinov — APPROVED** [review]

File: contracts/leverage_trading/src/metadata.rs (hunk 1 above)
Comment (vlad-sundunchan) [reply contracts/leverage_trading/src/metadata.rs]: Done

**Review by TymofiiSavinov — APPROVED** [review]

**Pull request merged**